verilog/parser/BUILD

```diff
@@ -117,11 +122,17 @@ record_recovered_syntax_errors(
 cc_library(
     name = "verilog_y_cc",
     srcs = ["verilog-final.tab.cc"],
-    hdrs = ["verilog_parse_interface.h"],
-    copts = [
-        "-Wno-implicit-fallthrough",
-        "-Wno-type-limits",
-    ],
+    hdrs = ["verilog_parse_interface.h"] + select({
+        "@platforms//os:windows": ["verilog.tab.hh"],
```

Comment:
So this is intentional to not include verilog.tab.hh, but the verilog_parse_interface which is generated.
What was the compile trouble you had seen when _not_ adding `verilog.tab.hh` here ?

Reply:
Without `verilog.tab.hh`, bazel complains `verilog-final.tab.cc` needs it and refuse to compile it.

I checked the generated `verilog.tab.cc` on Linux and it has this wonderful comment:

``` 
/* In a future release of Bison, this section will be replaced
   by #include "verilog.tab.hh".  */
```

And, of course, `win_flex_bison` has a "future release of Bison". I suggest I change the `win_flex_bison` release to match `rules_bison` version?

Reply:
Yes, I think for now matching the versions if possible would be best.
We then can later see how we can deal with this particular issue separately.